app/components/provider_interface/user_permission_summary_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= tag.dl(class: 'govuk-summary-list') do %>
+  <% ProviderPermissions::VALID_PERMISSIONS.each do |permission| %>
+    <%= tag.div(class: 'govuk-summary-list__row') do %>
+      <dt class="govuk-summary-list__key">
+        <%= description_for(permission) %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <%= can_perform_permission_y_n?(permission) %>
+        <% if ProviderRelationshipPermissions::PERMISSIONS.include?(permission) %>
+          <p>This user permission is affected by organisation permissions.</p>
+
+          <%= render ProviderInterface::ProviderPartnerPermissionBreakdownComponent.new(provider: provider, permission: permission) %>
+        <% end %>
```

Comment:
This block should not render if the answer to the above bit is "No". Can this check be added to the if?

Reply:
Ah ok. I did not see this specified anywhere 🤔

Reply:
So is it not possible to not be able to do something for your org, but to be able to do it for another org?

Reply:
This component will only be rendered in the context of a single provider. If a user belongs to multiple organisations then they will have a different page under that provider to show their permissions for that org.

E.g. they can make_decisions for provider A, and cannot make_decisions for provider B, but belongs to both.
If they go to Provider A > Users > Specific User, then they'll see: "Yes, this applies to courses..."
If they go to Provider B > Users > Specific User, then they'll just see "No" because org B does not let them do anything on any courses

Reply:
My bad with the requirements, I've managed to erase a whole batch which has made things very confusing

Reply:
Should `This user permission is affected by organisation permissions.` also be moved to the `ProviderPartnerPermissionBreakdownComponent.new component` then, as I assume we don't want to render that text on it's own if the user is able to perform a permission on behalf of an org but is not associated with any other providers. Having that outside that component means that we need to start pulling out partner providers at this level as well which is not ideal

Reply:
Agreed that we'll keep it the same as it is in the prototype (which is simple and requires the least change). We'll flag it up in a demo before turning the flag on